packages/ui-components/src/components/single-select-dropdown/single-select-dropdown.types.ts

```diff
@@ -39,27 +31,23 @@ export interface ISingleSelectDropdown extends ICustomCss {
 	helpText?: string | string[];
 	/** (optional) If `true` the dropdown is disabled */
 	disabled?: boolean;
-	/** (required) The text to display when there are no options */
-	noDataAvailableLabel?: string;
-	/** (optional) The object with the dropdown options */
-	options?: ISingleSelectDropdownOptions;
 	/** (optional) The value of the selected option */
 	selectedOption?: string;
-	/** (optional) The object with the dropdown options filtered */
-	filteredOptions?: ISingleSelectDropdownOptions;
-	/** (optional) The dropdown's min-height */
-	minHeight?: string;
-	/** (optional) The dropdown's max-height */
-	maxHeight?: string;
 	/** (optional) The size of the input */
 	inputSize?: EComponentSize;
 	/** (optional) The dropdown position config options */
 	dropdownOptions?: Partial<ComputePositionConfig>;
-	/** (optional) The minimum amount of options required to display the search. Defaults to `8`. */
-	minSearchOptions?: number;
+	/** (optional) If `false` clicking outside the dropdown will not trigger state change. Default: true */
+	clickOutsideClose?: boolean;
+	/** (optional) A reference to the dropdown action element */
+	actionElement?: HTMLElement;
+	/** (optional) The object with the dropdown options */
+	options?: ISelectSingleOptions;
+	/** (optional) The object with the dropdown options filtered */
+	filteredOptions?: ISelectSingleOptions;
 }
 
-export interface ISingleSelectDropdownEvents {
+export interface ISingleSelectDropdownEvents extends Omit<ISelectMultiOptionsEvents, 'optionsSelected' | 'selectAll'> {
```

Comment:
should we extend and omit from ISelectEvents instead?

Reply:
I don't think so otherwise we will need to redefine `optionSelected` that is only present in `ISelectMultiOptionsEvents`.
Nevertheless, after taking a look at this, I noticed there were some unnecessary property definitions. 
I'll remove them 👍